muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/organization/OrganizationJsPluginMessages_fi.properties

```diff
@@ -1,4 +1,10 @@
 plugin.organization.pageTitle = Organisaation hallinnointi
+plugin.organization.create.student.success = Opiskelijan luonti onnistui
+plugin.organization.create.student.error = Opiskelijan luonti ep�onnistui
+plugin.organization.create.staffmember.success = Opiskelijan luonti onnistui
```

Comment:
Staffmember should be something else than **opiskelija**. Maybe use **Henkilökunnan jäsenen** instead?